src/smpp_gateway/client.py

```diff
@@ -59,29 +63,37 @@ class PgSmppClient(smpplib.client.Client):
     https://gist.github.com/pkese/2790749
     """
 
-    def __init__(self, *args, **kwargs):
-        self.backend = kwargs.pop("backend")
-        self.submit_sm_params = kwargs.pop("submit_sm_params")
```

Comment:
If we're `pop()`ing these kwargs with no default, they are required.  Make this behavior obvious by having them as positional args.